### ISSUE TITLE
Xcode 8 GM Ctrl+W to switch window does not work

### ISSUE THREAD

**Issue by dacoltforty5** [XVimProject/XVim#989]
In Xcode 8 GM, typing Ctrl+W does not let me switch in between windows. Instead Xcode hangs and won't let me continue until I hit the Esc key.

**Comment by squarefrog** [comment]
Same behaviour here. As a work around do `cmd+j`

**Comment by hgonzalez94** [comment]
Yeah, same thing here :( my heart, it is broken. This is such a crucial part of my work flow. Paradise lost.
